### PR TITLE
Port fixes from learn guide

### PR DESCRIPTION
**Pull request by robin-norwood**

### CODE REVIEW

**Comment by judithpatudith** [comment]
👋 hey should I wait to test this till the Learn PR merges? And also , forgive my ignorance about Google Cloud Shell, but is there any way for me to preview this PR in a local build or staging environment? I'm a horrible proof reader, so if anything is up with the formatting I'll never find it in markdown 😜

**Comment by robin-norwood** [comment]
Unfortunately, GCP is the only way to preview it. Waiting until the other one is merged is probably a good idea, though.

**Comment by robin-norwood** [comment]
@tr0njavolta @judithpatudith - oops, I forgot about this one. :) These should be essentially equivalent to the changes for https://github.com/hashicorp/learn/pull/1076

**Review by judithpatudith — APPROVED** [review]

**Pull request merged**
